docs/dx.rst

```diff
@@ -0,0 +1,246 @@
+.. _dx:
+
+DNAnexus
+========
+
+DNAnexus paths on stor are prefixed with ``dx://`` and have two components:
+``dx://<PROJECT>:<FILE_OR_FOLDER>``
+where project and file can be virtual paths (i.e., human names) or canonical paths
+(opaque globally unique IDs that the platform assigns) - see below for more details.
+
+Canonical Paths on DNAnexus
+---------------------------
+Every file resource on DNAnexus is given a canonical ID to refer it by. This is
+internally given by the platform and is not determined by stor. However, stor
+does provide the option of referring to the resource through their canonical IDs.
+Each individual project and file has a canonical ID associated with it. This
+ID consists of 24 case-sensitive characters from ``0123456789BFGJKPQVXYZbfgjkpqvxyz``
+prepended with 'project-' or 'file-' respectively. Example canonical IDs are
+``project-j47b1k3z8Jqqv001213v312j1``, and ``file-47jK67093475061g3v95369p``.
+
+Each individual file with its unique ID, can be present only at a single location within
```

Comment:
delete this paragraph. we don't really know why they can only be at a single location.

Better to say, `DNAnexus only allows one copy of a file to be in a specific project` than the rest of this sentence

Reply:
Fixed